### PR TITLE
CTCTRADERS-2213: add route for non EU countries list

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: app/data/transform/TransformationImplicits.scala [hunk 1]
```diff
@@ -49,7 +50,21 @@ trait TransformationImplicits {
           )
       )
 
-  implicit val transformationCountryCodesCommonTransitList: Transformation[CountryCodesCommonTransitList.type] =
+  implicit val countryCodesCommonTransitOutsideCommunityList: Transformation[CountryCodesCommonTransitList.type] =
```
Comment (MatthewAlexanderBeck) [inline app/data/transform/TransformationImplicits.scala:53]: Looks like the return types are mixed up here

File: app/data/transform/TransformationImplicits.scala [hunk 2]
```diff
@@ -49,7 +50,21 @@ trait TransformationImplicits {
...
+            (__ \ Common.state).json.pickBranch and
+            (__ \ Common.activeFrom).json.pickBranch and
+            (__ \ Common.description).json.copyFrom(englishDescription)
+        ).reduce
+          .andThen(
+            (__ \ Common.activeFrom).json.prune
+          )
+      )
+
+  implicit val transformationCountryCodesCommonTransitList: Transformation[CountryCodesCommonTransitOutsideCommunityList.type] =
```
Comment (MatthewAlexanderBeck) [inline app/data/transform/TransformationImplicits.scala:67]: And here

File: app/data/transform/TransformationImplicits.scala (hunk 1 above)
Comment (kevincoatsworth) [reply app/data/transform/TransformationImplicits.scala]: :+1:

File: app/data/transform/TransformationImplicits.scala (hunk 2 above)
Comment (kevincoatsworth) [reply app/data/transform/TransformationImplicits.scala]: :+1:

**Review by MatthewAlexanderBeck — APPROVED** [review]

**Pull request merged**